server/routes/import_wizard/html.py

```diff
@@ -30,3 +31,58 @@ def main():
 @bp.route('/')
 def main_new():
   return render_template('/import_wizard2.html')
+
+
+# This is only used in CSV + SQL backend, converting raw CSV data into resolved
+# CSV.
+@bp.route('/simple/convert', methods=['POST'])
+def convert():
+  RAW_DATA_PATH = os.environ.get("RAW_DATA_PATH")
+  SQL_DATA_PATH = os.environ.get("SQL_DATA_PATH")
+  command1 = [
+      "python",
+      "import/simple/stats/main.py",
+      "--input_path",
+      f"{RAW_DATA_PATH}/input.csv",  # TODO: use gcs and local folder path.
```

Comment:
Once the script supports folders with a config.json, I'm assuming we'll change this as well?